src/pdbvangogh/api.py

```diff
@@ -0,0 +1,43 @@
+from pdbvangogh import pdb
+from pdbvangogh.img import *
+from pdbvangogh.models import *
+from pdbvangogh.data_models import *
+import os
+
+
+def test_func():
+    return True
+
+
+def pdbvangogh(
+    background_image,
+    style_image,
+    save_prefix,
+    out_dir=None,
+    content_image=None,
+    pdb_id=None,
+    background_size = 500,
+    content_size = 800, 
+    background_hyperparameters=vgg19_transfer_parameters(style_weight=1e-2, epochs=10, steps_per_epoch=100),
+    content_hyperparameters=vgg19_transfer_parameters(style_weight=1e-4, epochs=10, steps_per_epoch=100),
+    debug=False,
+):
```

Comment:
you may want to add a docstring here if this is the top-level api describing the parameters, etc..  google style or sphinx are popular formats